modules/mobile/spec/request/v1/user_request_spec.rb

```diff
@@ -233,200 +234,6 @@
           )
         end
       end
```

Comment:
This is a major benefit of moving authorized services out of the serializer. We're able to unit test, so we don't have to have these clunky tests anymore.